sdk/servicebus/service-bus/test/README.md

```diff
@@ -4,8 +4,6 @@
 - The tests expect a Service Bus namespace to exist.
   For Node environment, the connection string for the service bus should be in the environment variable `SERVICEBUS_CONNECTION_STRING`.
 
-  For browser environment, the connection string for the service bus should be in the environment variable `SERVICEBUS_CONNECTION_STRING_BROWSER`.
```

Comment:
Now that a separate note for browser is not needed, the note above for Node needs to be generic and not refer to "node environment"